src/processHistorian.py

```diff
@@ -112,11 +110,6 @@ def __init__(self):
         print("Work threads:")
         print(self.__threads)
 
-        # Eighth step: Subscribe all data and start all threads
-        self._opcua_client.subscribe_all()
```

Comment:
Im not sure if it's a good idea to not setup correnctly but wait for an error

Reply:
Suppose the watchdog works - we'd then have a structure which deals well with "no connection"-States - Startup really is just a special form of no-connection
The alternative would be to replicate most of the functionality of the watchdog just for the initial connection.